precompiles/proxy/src/lib.rs

```diff
@@ -200,7 +200,7 @@ where
 		}
 		.into();
 
-		<RuntimeHelper<Runtime>>::try_dispatch(handle, Some(origin).into(), call)?;
+		<RuntimeHelper<Runtime>>::try_dispatch(handle, Some(origin).into(), call, 0)?;
```

Comment:
I know that this change comes from a [cherry-pick](https://github.com/Moonsong-Labs/moonkit/commit/a32f7577b2b2a877645b7369bb5c90112e6038ad), could we take the opportunity to use proper storage costs?

Reply:
Addressed in https://github.com/Moonsong-Labs/moonkit/pull/59/commits/3f61599b0716dd77a78cd761589f098f023c9d9d